### PR TITLE
Created Homepage, along with nav bar, you can navigate to different p…

### PR DESCRIPTION
**Pull request by emc3424**
…ages, and look at a brief hard coded table of recent activity. There is an issue with using bootstrap charts

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,20 +2,24 @@ import logo from "./logo.svg";
...
-      </header>
-    </div>
-  );
+       {/* <header className="App-header"> */}
+         {/* <img src={logo} className="App-logo" alt="logo" />
+    //     <p>
+    //       Edit <code>src/App.js</code> and save to reload.
+    //     </p>
+    //     <h1>Hey</h1>
+    //     <Button variant="orange">Test</Button> */}
```
Comment (emrusz) [inline src/App.js]: Please remove template comments if they are no longer needed. (The variables defined in them are setting off eslint.)

image

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </nav>
+    )
+}
+
+export default Navbar
```
Comment (emrusz) [inline src/components/Navbar.js]: Please add a trailing newline.

File: src/pages/Activitylog.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+function ActivityLog() {
+    return (
+        <div className="ActivityLog">
+            <Navbar />
+            <h1>ActivityLog</h1>
+        </div>
+    );
+}
+
+export default ActivityLog;
```
Comment (emrusz) [inline src/pages/Activitylog.js]: Please add a trailing newline.

File: src/pages/Classes.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+function Classes() {
+    return (
+        <div className="Classes">
+            <Navbar />
+            <h1>Classes</h1>
+        </div>
+    );
+}
+
+export default Classes;
```
Comment (emrusz) [inline src/pages/Classes.js]: Please add a trailing newline.

File: src/styles/navbar.css [hunk 5]
```diff
@@ -0,0 +1,7 @@
+.navbar {
+    background-color:rgb(224, 111, 45);
+}
+
+.nav-link{
+    color: white;
+}
```
Comment (emrusz) [inline src/styles/navbar.css]: Please add a trailing newline.

**Review by emrusz — CHANGES_REQUESTED** [review]
I've requested some changes, mostly to do with code styling and one for some unused vars. Besides that, it looks good and runs as expected in the container.

**Review by JordanMRabideau — APPROVED** [review]

**Review by emrusz — APPROVED** [review]

**Pull request merged**